examples/factory-contract/Cargo.toml

```diff
@@ -12,6 +12,7 @@ serde_json = "1.0"
 test-case = "2.0"
 tokio = { version = "1.14", features = ["full"] }
 near-workspaces = { version = "0.8.0" }
+near-token = { version = "0.1.0", features = ["serde", "borsh", "schemars"] }
```

Comment:
Use `near_workspaces::types::NearToken` instead of importing near-token separately